opensrp-app/src/main/java/org/smartregister/sync/helper/LocationServiceHelper.java

```diff
@@ -405,6 +411,9 @@ public void fetchAllLocations() {
             JSONObject request = new JSONObject();
             request.put(IS_JURISDICTION, true);
             request.put(AllConstants.SERVER_VERSION, 0);
+            if (parentId != null) {
```

Comment:
Could we use `StringUtils.isNotBlank(parentId)` instead?

Reply:
fixed.